app/views/posts/show.html.erb

```diff
@@ -1,23 +1,26 @@
-<% stylesheet_link_tag "tables"%>
+<% stylesheet_link_tag "posts-list"%>
 
+<main class='main-container'>
 <h1>Specific Post</h1>
+<section class='post-extensive'>
+  <div class='post-container'>
+    <div class='post-header'>
+      <h3 class='post-title'><%= @post.title %> (Post #<%= @post.id %>)</h3>
+      <span class='likes-and-comments'>
+        Comments: <%= @post.comments_counter %>, Likes: <%= @post.likes_counter %>
+      </span>
+    </div>
+    <p class='post-text'>
+      <%= @post.text %>
+    </p>
+  </div>
+  <div class='post-list-comments'>
+    <% @comments.each do |comment| %>
+      <%= render partial: 'shared/comment-container', locals: { comment: comment, user: @user } %>
+    <% end %>
+  </div>
```

Comment:
- [ ] Post show page should display post with all comments listed as shown on the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_single_post.png)Post show page displays post with all comments listed as shown on the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_single_post.png). Currently, I'm not able to test it in the views of the app